### PR TITLE
Feature use env

### PR DESCRIPTION
**Pull request by rubenrua**

### CODE REVIEW

File: app/config/config.yml [hunk 1]
```diff
@@ -32,8 +55,9 @@ twig:
     debug:            "%kernel.debug%"
     strict_variables: "%kernel.debug%"
     globals:
-        pumukit2_info: "%pumukit2.info%"
-        locales:       "%pumukit2.locales%"
+        pumukit_info:  "%pumukit.info%"
+        pumukit2_info: "%pumukit.info%"
```
Comment (Yurujai) [inline app/config/config.yml]: why keep pumukit2_info parameter?

**Comment by rubenrua** [comment]
ping

File: app/config/config.yml (hunk 1 above)
Comment (Yurujai) [reply app/config/config.yml]: Done https://github.com/campusdomar/PuMuKIT2/pull/1074/commits/ed0ed87fae80ec731205a2040aadafa5dd68d1cd

File: app/config/parameters.yml.dist [hunk 2]
```diff
@@ -1,31 +1,22 @@
...
-    mongodb_server:    mongodb://localhost:27017
-    mongodb_database:  pumukit
+    env(MONGODB_SERVER): 'mongodb://localhost:27017'
+    env(MONGODB_DATABASE): pumukit
+    env(MAILER_TRANSPORT): smtp
+    env(MAILER_HOST): 127.0.0.1
+    env(MAILER_USER): null
+    env(MAILER_PASSWORD): null
+    env(LOCALE): en
+    env(SECRET): ThisTokenIsNotSoSecretChangeIt
```
Comment (j-lorenzo) [inline app/config/parameters.yml.dist]: Why you don't apply the prefix "PUMUKIT" to all environment variables?

Comment (Yurujai) [reply app/config/parameters.yml.dist]: Done https://github.com/campusdomar/PuMuKIT2/pull/1074/commits/340e7299cbcb97e28b604051e260aa76b2b5a3d8

**Pull request merged**